### DISCUSSION TITLE
Files unstable after snapshot taken

### DISCUSSION THREAD

**Discussion by widders** [Altinity/clickhouse-backup#243] [Q&A] [answered]
We have a pair of nodes running ReplicatedMergeTree tables approaching 2TB. Using clickhouse-backup we take daily backups along with 6 hourly snapshots on one of the nodes. We use an external process to backup the files and it takes between between 24-30 hours so 4 or 5 snapshots and the next daily backup are created during the backup process but this has been taking into account with the retention of the snapshots.

During the process tar reports that several files were edited during read, they tend to be on our heavily used tables but can be on partitions up to a year old. My understanding was that once a snapshot is created and the parts frozen no further writes should take place within the /var/lib/clickhouse/backups/<name>/ directory. We have sync replicas enabled in the clickhouse-backup config, is this process asynchronous and not blocking to the termination of clickhouse-backup? Or is taking further snapshots potentially causing writes to the original files?

From what I can tell both clickhouse and the file system see the last modification to the part being days before the snapshot was even made. Has anyone seen a similar issue with taking backups like this?

**Comment by Slach** [comment, marked as answer]
Hmm. Look weird, your understanding is correct. After creating backup, writes to /var/lib/clickhouse/backup/backup_name unexpected

how you detect "writes" do you see new files or in /var/lib/clickhouse/backup/backup_name  ?
or you just track file size for this folder like `du -s -h /var/lib/clickhouse/backup/backup_name` ?

Did you try last version of clickhouse-backup 1.4.7? Effect still present?
